controllers/servers/csi/addons_server.py

```diff
@@ -156,3 +157,14 @@ def _ensure_replication_idempotency(replication_request, replication, volume):
                       "but volume {} belongs to another group {}".format(volume.name, volume.volume_group_name)
                 return error_message
         return None
+
+    @staticmethod
+    def _get_and_validate_volume(mediator, volume_id):
+        volume = mediator.get_object_by_id(volume_id, servers_settings.VOLUME_TYPE_NAME)
+        if not volume:
+            raise array_errors.ObjectNotFoundError(volume_id)
+        if volume.volume_group_id:
+            error_message = "could not enable replication, " \
+                      "volume {} already belongs to volume group {}".format(volume.name, volume.volume_group_id)
+            return volume, error_message
+        return volume, None
```

Comment:
```suggestion
        return volume, ""
```

Reply:
I'd rather prefer to return None as no-error-message. Especially because the function _ensure_replication_idempotency returns None in the similar case. And I want to be consistent